aspnetcore/performance/caching/memory.md

```diff
@@ -86,6 +96,37 @@ The following sample:
 
 [!code-csharp[](memory/sample/WebCache/Controllers/HomeController.cs?name=snippet_et&highlight=14-21)]
 
+::: moniker range=">= aspnetcore-2.0"
+
+## Use SetSize, Size, and SizeLimit to limit cache size
+
+A MemoryCache instance may optionally specify and enforce a size limit. The memory size limit does not have a defined unit of measure because the cache has no mechanism to measure the size of entries. If the cache memory size limit is set, then all entries must specify size. The size specified is in units the developer chooses.
+
+For example:
+
+* If the web app was primarily caching strings, each cache entry size could be set to the string length.
+* The app could specify the size of all entries as 1, and the size limit is the count of entries.
+
+The following code creates a unitless fixed size [MemoryCache](/dotnet/api/microsoft.extensions.caching.memory.memorycache?view=aspnetcore-2.1) accessible by [dependency injection](xref:fundamentals/dependency-injection):
```

Comment:
The first link seems to lead to a 404

Reply:
Nevermind, I see that this is available on docs.microsoft.com but not on review.docs.microsoft.com.